src/DynamoManipulation/DynamoManipulation.csproj

```diff
@@ -42,6 +42,10 @@
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.4074" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="$(PkgDynamoVisualProgramming_LibG_230_0_0)\tools\netstandard2.0\windows\LibG_230_0_0\Autodesk.GeometryPrimitives.Dynamo.dll">
```

Comment:
why do we need to reference this directly when we have the reference to libg230 above already? That only references protogeometry.dll for some reason?

Reply:
This is in `tools` subfolder, which does not automatically get referenced.